lib/index.js

```diff
@@ -166,8 +166,13 @@ else if (typeof window !== 'undefined') {
  * // creating a stream from events
  * _('click', btn).each(handleEvent);
  *
- * // creating a stream from events with mapping
+ * // creating a stream from events with a mapping array
  * _('request', httpServer, ['req', 'res']).each(handleEvent);
+ * //=> { req: IncomingMessage, res: ServerResponse }
+ *
+ * // creating a stream from events with a mapping function
+ * _('request', httpServer, function(req, res) { return res; }).each(handleEvent);
```

Comment:
Looks good. Can you line wrap the anonymous function so it's not all on one line?